src/main/java/frc/robot/Controls.java

```diff
@@ -134,11 +144,19 @@ private void configureDrivebaseBindings() {
                       .withRotationalRate(newSpeeds.omegaRadiansPerSecond);
                 })
             .withName("Drive"));
-    s.drivebaseSubsystem
-        .applyRequest(() -> brake)
-        .ignoringDisable(true)
-        .withName("Brake")
-        .schedule();
+    RobotModeTriggers.disabled()
+        .onTrue(
+            s.drivebaseSubsystem
+                .applyRequest(() -> brake)
+                .ignoringDisable(true)
+                .withName("Disable Brake"));
```

Comment:
? This sets the robot in brake mode, so either the comment is wrong or this is calling the wrong command.

Reply:
If the comment is wrong, I think this also won't do anything until the robot becomes enabled since motors can't be actuated during disabled mode, so the wheels won't move to the X position until the robot is enabled (and assuming nothing else overrides the swerve command set here before the robot is enabled).